libs/frontend/feature-template/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "frontend-feature-template",
+  "name": "frontend-feature-template-builder",
```

Comment:
I added the `-builder` suffix to the `name` of some of the `project.json` in `pubstudio/pubstudio-builder` 
 because there are some project name conflicts between `pubstudio/libs` and `pubstudio/pubstudio-builder/libs`, otherwise Nx will throw an error when executing commands like `npx nx serve web`.

![Screenshot 2024-01-23 at 3 44 59 PM](https://github.com/samatechtw/pubstudio-builder/assets/59676941/bcb81946-607d-4359-9640-af3f25ed2b72)

Reply:
That should be fine! In theory we should also update the folder name and references, otherwise some NX commands might get confused. But in this case I think there's a low chance we'll need to use NX commands on the library, so this is a good solution.